### PR TITLE
Latest working iteration of Hellosign NodeJS Demo

### PR DESCRIPTION
**Pull request by irwynks**
This is the finalized version of the demo. All files have been replaced to ensure no conflicts.

### CODE REVIEW

**Comment by cmpaul** [comment]
@radhack Would you give this a review as well, specifically around the documentation and configuration. Thanks!

File: config.js [hunk 1]
```diff
@@ -1,9 +1,14 @@
-module.exports = {
+var config = {
+    mongodb:{},
+    hellosignKey: process.env.HELLO_KEY || '40b740615f6f87a49c72f6beefd4c880d476ee1c58994955a42bf33ad596e3b2',
```
Comment (radhack) [inline config.js]: We're going to remove this key from the repo and reset the API Key on your account - @irwynks you'll need to pull and update your environment with your new key (which you can get from the hellosign.com account API tab).

File: config.js [hunk 2]
```diff
@@ -1,9 +1,14 @@
-module.exports = {
+var config = {
+    mongodb:{},
+    hellosignKey: process.env.HELLO_KEY || '40b740615f6f87a49c72f6beefd4c880d476ee1c58994955a42bf33ad596e3b2',
+    hellosignID: process.env.HELLO_ID || '94cd46c0b0f52e25225f4406bdaa2b07'
```
Comment (radhack) [inline config.js]: We'll also remove this client_id

File: views/partials/embedded-signing.handlebars [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        type:'POST',
+        url:'/getEmbeddedSigning',
+        success: function(response) {
+
+            console.log(response);
+
+            HelloSign.open({
+                url: response.embedded.sign_url,
+                container:document.getElementById('embeddedSigningContainer'),
+                allowCancel: true,
```
Comment (radhack) [inline views/partials/embedded-signing.handlebars]: We need to add uxVersion: 2, here

**Pull request merged**